tech/tech/interfaces/schemas/order_schema.py

```
@@ -0,0 +1,36 @@
+from pydantic import BaseModel
+from enum import Enum
+
+class OrderStatusEnum(str, Enum):
```

Ruff (0.8.2)

Expected 2 blank lines, found 1

Add missing blank line(s)

(E302)